### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by ealcobaca**
Setup
-------

* setup file
* version file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"""Setup for paje-data package."""
+import os
+import setuptools
+import pjdata
+
+NAME = 'paje-data'
+
+
+VERSION = pjdata.__version__
```
Comment (davips) [inline setup.py]: esagsgsrh

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+AUTHOR = 'Davi Pereira dos Santos'
+
+
+AUTHOR_EMAIL = ''
+
+
+DESCRIPTION = 'Pajé Data'
+
+
+with open('README.md', 'r') as fh:
```
[the remark this answers is not in the document]
Comment (ealcobaca) [reply setup.py]: > wtf

:)

**Comment by ealcobaca** [comment]
> unacceptable

why? hehe

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+"""Setup for paje-data package."""
+import os
+import setuptools
+import pjdata
+
+NAME = 'paje-data'
```
Comment (davips) [inline setup.py]: last wtf

Comment (ealcobaca) [reply setup.py]: I will change the quotes by double quotes ;D

**Comment by davips** [comment]
beautiful

**Comment by davips** [comment]
beautiful

**Pull request merged**